### PR TITLE
Fix/getter setter

### PR DESCRIPTION
**Pull request by kurtommy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -947,6 +947,35 @@ const bankAccount = new BankAccount();
 // Buy shoes...
 bankAccount.withdraw(100);
 ```
+
+**Good with getter/setter**:
```
Comment (ryanmcdermott) [inline README.md]: Let's just keep everything either Bad or Good

File: README.md [hunk 2]
```diff
@@ -947,6 +947,35 @@ const bankAccount = new BankAccount();
...
+```javascript
+class BankAccount {
+  constructor(balance = 1000) {
+    this_balance = balance;
```
Comment (ryanmcdermott) [inline README.md]: Fix `this_balance`

File: README.md [hunk 3]
```diff
@@ -947,6 +947,35 @@ const bankAccount = new BankAccount();
...
+  }
+
+  // It doesn't have to be prefixed with `get` or `set` to be a getter/setter
+  set balance(amount) {
+    if (verifyAmountCanBeSetted(amount)) {
```
Comment (ryanmcdermott) [inline README.md]: While you're at it, can you write a fake method for `verifyAmountCanBeSettled`? It doesn't have to have any code other than `// ...` Thanks!

**Comment by vsemozhetbyt** [comment]
Will fix https://github.com/ryanmcdermott/clean-code-javascript/issues/45

**Comment by ryanmcdermott** [comment]
Cool I love it!

**Pull request merged**